script/cibuild

```diff
@@ -2,16 +2,16 @@
 
 set -e
 
-. script/bootstrap
+. script/common
 
 echo "Verifying code is formatted via 'gofmt -s -w  go/'"
-gofmt -s -w  go/
-git diff --exit-code --quiet
+$DGO gofmt -s -w go
+git diff --exit-code --quiet go
 
-echo "Building"
-script/build
-
-cd .gopath/src/github.com/github/gh-ost
+if [ "$1" != "fast" ]; then
```

Comment:
If you just want to run gofmt and test, you can run this as `./script/cibuild fast`.

However, see my top-level comment. Something feels off that this is separate from `build.sh`. I'm thinking that this code should just be combined into one script at this point.